docs/execution/WorkflowExecution.md

```diff
@@ -0,0 +1,96 @@
+This section explains at a high level how Cromwell runs a workflow, and how workflow completeness is determined.
+
+In order to run a workflow, Cromwell uses the backends available to it to create jobs and monitor them until they are complete. When no new jobs can or should be started, and all jobs have reached a terminal status (Success, Failure, Aborted), the workflow itself is terminated.
+
+In a ideal situation, all jobs succeed and the workflow succeeds.
```

Comment:
The doc is called "workflow execution" but 99% of it appears to be "something has gone horribly awry". Not a good message to be sending :)

Reply:
hmm that's true, any suggestion ? :)

Reply:
If I had a good one I'd have made it .... "when good workflows go bad"? :)

Reply:
just keepin' it real